modules/client-configuration/nav-client-config-guide.adoc

```diff
@@ -104,6 +111,19 @@ include::modules/client-configuration/pages/virt-vmware.adoc[leveloffset=+2]
 
 include::modules/client-configuration/pages/virt-file.adoc[leveloffset=+2]
 
+// VHMs
+include::modules/client-configuration/pages/vhm.adoc[leveloffset=+1]
+
+include::modules/administration/pages/vhm-azure.adoc[leveloffset=+2]
```

Comment:
Link is wrong, this page is under modules/client-configuration/pages/vhm-azure.adoc
Is that the right place?